config/F16.config

```diff
@@ -3,7 +3,37 @@ process {
         cpus = 1
         memory = 250.MB
     }
-    withName: run_stats_SAMtools {
+    withName: assess_ReadQuality_FastQC {
+        cpus = 2
+        memory = 1.GB
+        retry_strategy {
+            memory {
+                strategy = 'add'
+                operand = 4.GB
+            }
+        }
+    }
+    withName: run_stats_SAMtools_readgroup {
```

Comment:
**(question|standardization : non-blocking)** Does this follow our NF standards?

Reply:
I believe so.  The standards say:
"It’s ok to add an additional attribute if a process needs to be run more than once"

`run_stats_SAMtools` is the base process name.

Reply:
>"It’s ok to add an additional attribute if a process needs to be run more than once"

Right. I think placing the tool name as the last attribute would make more sense though (e.g. run_VariantRecalibratorSNP_GATK and run_VariantRecalibratorINDEL_GATK)? I'll defer this to Nextflow WG.
https://uclahs-cds.atlassian.net/wiki/spaces/BOUTROSLAB/pages/3193890/Nextflow+pipeline+standardization

Reply:
Oh I see!  What are your thoughts on whether the three sections should be maintained?  `run_statsReadgroups_SAMtools` vs `run_stats_readgroups_SAMtools`

Reply:
I'm ok with maintaining the three sections and using `run_statsReadgroups_SAMtools`